dev/refresh_source.py

```diff
@@ -434,7 +434,8 @@ def write_deck_run_script(self, repository):
 cd "$d"
 LOG_DIR=${{LOG_DIR:-../logs}}
 
-bash -c "(./start.sh >> '$LOG_DIR/{name}.log') 2>&1\
+npm install >& "$LOG_DIR/deck.log"
```

Comment:
I can confirm that modifying `start_dev.sh` to contain what this routine will generate does result in deck starting up properly on a dev vm.
It does not appear that `start.sh` is used anywhere.